src/metabase/api/metric.clj

```diff
@@ -25,6 +26,13 @@
   (check-superuser)
   (check-404 (metric/retrieve-metric id)))
 
+(defendpoint GET "/"
+  "Fetch all `Metrics`. You must be a superuser to do this."
+  [id]
+  (check-superuser)
```

Comment:
All the other endpoints to get metrics & segments require superuser status so I was just going along with that ?